src/imageio/format/jpeg.c

```diff
@@ -489,6 +563,7 @@ void gui_reset(dt_imageio_module_format_t *self)
 {
   dt_imageio_jpeg_gui_data_t *g = (dt_imageio_jpeg_gui_data_t *)self->gui_data;
   dt_bauhaus_slider_set(g->quality, dt_confgen_get_int("plugins/imageio/format/jpeg/quality", DT_DEFAULT));
+  dt_bauhaus_combobox_set(g->subsample, 0);    // auto
```

Comment:
It might be a good idea to still add the entry w/ a default value to `darktableconfig.xml.in` (and use it here)?

Reply:
What problem would it solve?

Reply:
> What problem would it solve?

Long term maintenance.

The "failing" `dt_conf_get_int()` from existing/legacy user .rc files without this setting present, without making (undocumented) assumptions the fallback value is 0 (i.e. auto; as sensible as it is).

A possibility for a setting up a default other than "auto" (0).

Reply:
> A possibility for a setting up a default other than "auto" (0).

Default must never be other than "auto", so it is not an argument.

Reply:
> Default must never be other than "auto"

Why not? Where is it documented?